stories/MainSearch.stories.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { action } from '@storybook/addon-actions';
+import MainSearch from '../src/components/main-search';
```

Comment:
I liked how we used to do `import { MainSearch } from '../src/components';` (from the index file) because this also tested that we had exported it properly thus preventing the numerous `cannot find...` errors when attempting to import from uniprot.